glyphs-reader/src/font.rs

```diff
@@ -40,13 +43,32 @@ pub struct Font {
     pub axis_mappings: BTreeMap<String, RawUserToDesignMapping>,
 }
 
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct Glyph {
+    pub glyphname: String,
+    pub instances: Vec<GlyphInstance>,
```

Comment:
why did you name it `instances: Vec<GlyphInstance>` and not, say, `layers: Vec<Layer>`? I would expect this crate would stick to names that are close to those Glyphs.app itself uses

Reply:
I see you already have a type called `Layer`, maybe you can rename the latter `RawLayer` and rename `GlyphInstance` -> `Layer`

Reply:
then instead of `layer_to_glyph_instance` you could `impl TryFrom<RawLayer> for Layer`

Reply:
Oh I like that, updating.